### ISSUE TITLE
Feature: specify domain name as remote address

### ISSUE THREAD

**Issue by ttimasdf** [wangyu-/UDPspeeder#272]
Currently the `-r` parameter only accepts IPv4/v6 address. For a udpspeeder server hosted on OpenWrt, it's external IP address is changing regularly because of the restrictions from ISP. Maybe UDPSpeeder could take a domain name (updated by DDNS) and resolve (maybe cache) it regularly to resume the connection timely.

**Comment by wangyu-** [comment]
See https://github.com/wangyu-/UDPspeeder/wiki/ipv6-and-domain

**Comment by maurerr** [comment]
yes that can work temporally but what what about the server behind a dynamic ip - the domain would be the only solution here
